cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -60,7 +60,7 @@ object ReporterCommand : CommandWithHelp() {
     }
 
     @Parameter(
-        description = "The ORT result file to use. Must contain a scan result.",
```

Comment:
For the reporter I was going to write (will discuss about this with you and Martin before) soon will need the actual license texts. Just FYI. Also 3rd party implementation might want to use the scan result as input. 

I'm fine with this change anyway.

Reply:
Right, but then the individual reporter should fail, and not the general entry point (or say that a scan result is generally required).